index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav class="nav-bar">
+      <a href="/" class="logo">My Logo</a>
+      <img class="menu" src="/img/Union.svg " alt="" />
+    </nav>
+    <section class="headline">
+      <div class="container">
+        <h1>I’m Mohammed Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+      </div>
+      <div class="contact-me">
+        <p>LET'S CONNECT</p>
```

Comment:
- [ ] `Let's connect` is a call for action. I would recommend adding the `a` tag, so the user can click on it.